net-irc/irssi/irssi-1.0.0.recipe

```diff
@@ -7,31 +7,29 @@ HOMEPAGE="https://irssi.org/"
 COPYRIGHT="1999-2016 Timo Sirainen"
 LICENSE="GNU GPL v2"
 REVISION="1"
-srcGitRev="1cfec5f63d4adf32117f68638fb35802146e2784"
-SOURCE_URI="https://github.com/irssi/irssi/archive/$srcGitRev.tar.gz"
-CHECKSUM_SHA256="e4118fedeee70133b4aab70406c30be107cc0078c4d5c509e2b068277284d88a"
-SOURCE_DIR="irssi-$srcGitRev"
-PATCHES="irssi-ncurses.patch"
+SOURCE_URI="https://github.com/irssi/irssi/releases/download/$portVersion/irssi-$portVersion.tar.gz"
+CHECKSUM_SHA256="790eb3b62d8889d1fb3973845d8ca5d7c2578620632abc671ffdff3da13aa487"
+SOURCE_DIR="irssi-$portVersion"
 
-ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
 
 PROVIDES="
-  	irssi$secondaryArchSuffix = $portVersion
-  	cmd:irssi$secondaryArchSuffix
+	irssi$secondaryArchSuffix = $portVersion
```

Comment:
Since we don't have any SECONDARY_ARCHITECTURES we should drop every occurrence of $secondaryArchSuffix and ${secondaryArchSuffix}.

BTW, It's funny that nobody (including myself) has spotted this until now ;-)

BTW, you can also drop `cmd:chmod` from BUILD_PREREQUIRES, as chmod is in coreutils:
```
package list -p /system/packages/coreutils-*-*.hpkg | grep ^bin/chmod
```